packages/amplify-provider-awscloudformation/src/initialize-env.ts

```diff
@@ -20,7 +21,7 @@ async function run(context, providerMetadata) {
   const currentCloudBackendDir = context.amplify.pathManager.getCurrentCloudBackendDirPath();
   const backendDir = context.amplify.pathManager.getBackendDirPath();
 
-  const s3 = await S3.getInstance(context);
+  const cfnItem = await new Cloudformation(context);
```

Comment:
why was it moved if not used before?

Reply:
This is from leftover piece where I first did rollback with the pull command.